Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -3324,6 +3327,26 @@ void GraphicsContextGLANGLE::requestExtension(const String& name)
         m_webglColorBufferFloatRGB = true;
 }
 
+bool GraphicsContextGLANGLE::validateClearBufferv(GCGLenum buffer, size_t valuesSize)
+{
+    // This is missing ANGLE Robust* entry-point, so for consistency sake validate here
```

Comment:
`for consistency sake` sounds wrong to my ear. `for consistency's sake` sounds better.